docs/guide/replacement-guides/depcheck.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Modern alternatives to depcheck for analyzing project dependencies and unused code
+---
+
+# Replacements for `depcheck`
+
+## `knip`
+
+[knip](https://github.com/webpro-nl/knip) is a more actively maintained and feature-rich alternative to [`depcheck`](https://github.com/depcheck/depcheck).
+
+Example:
+
+```json
+{
```

Comment:
maybe since these configs wont really cross over, we should just show a basic knip config?

or point out that it usually doesn't need config